<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>iletisimEkrani</class>
 <widget class="QWidget" name="iletisimEkrani">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>2000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>760</y>
     <width>311</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 30pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Göster</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>620</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 25pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Ürün adı:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>1800</y>
     <width>311</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 30pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>İptal</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>610</y>
     <width>511</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>1250</y>
     <width>711</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 30pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>1100</y>
     <width>491</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 25pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Satıcının Bilgileri:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
